design-system/atoms/textarea/adapters/next-ui/next-ui.variants.ts

```diff
@@ -2,29 +2,30 @@ import { tv } from "tailwind-variants";
 
 export const TextareaNextUiVariants = tv({
   slots: {
+    container: "flex w-full flex-col gap-sm",
     base: "flex h-fit flex-col items-start gap-2",
-    mainWrapper: "",
-    inputWrapper:
-      "border-container-stroke-separator text-text-1 group-data-[focus=true]:border-interactions-white-hover group-data-[hover=true]:border-interactions-white-hover rounded-md border !outline-none",
+    inputWrapper: [
+      "transition-all",
+      "flex w-full items-stretch justify-start gap-md rounded-md border-1 border-border-primary bg-background-primary-alt effect-box-shadow-xs",
+      "data-[hover=true]:border-border-primary-hover hover:border-border-primary-hover",
+      "px-xl py-lg",
+      "group-data-[focus=true]:border-border-active group-data-[focus=true]:effect-ring-brand-glued",
```

Comment:
```suggestion
      "group-data-[focus-visible=true]:border-border-active group-data-[focus-visible=true]:effect-ring-brand-glued",
```

Reply:
focus-visible=true not working

Reply:
I can see it in the docs though https://nextui.org/docs/components/input#data-attributes 🤔

Reply:
Ah but I'm probably mixing this up with button focus states that have unwanted behaviour, you can forget about this.